starlette_admin/statics/js/actions.js

```diff
@@ -0,0 +1,63 @@
+function submitAction(name, form, customResponse) {
```

Comment:
I think you miss this for actions with customResponse
```suggestion
  $('a[data-no-confirmation-action="true"]').each(function () {
    $(this).on("click", function (event) {
      submitAction(
        $(this).data("name"),
        null,
        $(this).data("custom-response") === true
      );
    });
  });
  
function submitAction(name, form, customResponse) {
```

Reply:
Yes, Thank you!